README.md

```diff
@@ -31,13 +56,13 @@ L:   1 | P:  14 | L006 | Operators should be surrounded by a single space unless
 L:   1 | P:  27 | L001 | Unnecessary trailing whitespace
 ```
 
-You can also have a play using [SQLFluff online](https://online.sqlfluff.com/).
+You can also have a play using [**SQLFluff online**](https://online.sqlfluff.com/).
 
-For full CLI usage and rules reference, see the docs.
+For full [CLI usage](https://docs.sqlfluff.com/en/stable/cli.html) and [rules reference](https://docs.sqlfluff.com/en/stable/rules.html), see [the SQLFLuff docs](https://docs.sqlfluff.com/en/stable/).
 
 # Documentation
 
-For full documentation visit [docs.sqlfluff.com](https://docs.sqlfluff.com/en/stable/).
+For full documentation visit [docs.sqlfluff.com](https://docs.sqlfluff.com/en/stable/). This documentation is generated from this repository so please raise [issues](https://github.com/sqlfluff/sqlfluff/issues) or pull requests for any additions, corrections, or clarifications.
```

Comment:
```suggestion
For full documentation visit [docs.sqlfluff.com](https://docs.sqlfluff.com/en/stable). This documentation is generated from this repository so please raise [issues](https://github.com/sqlfluff/sqlfluff/issues) or pull requests for any additions, corrections, or clarifications.
```

Reply:
I disagree with this suggestion. Visiting https://docs.sqlfluff.com/en/stable explicitly redirects to the trailing slash version: https://docs.sqlfluff.com/en/stable/ so let's save users the effort of the redirect.